packages/terra-avatar/src/variants/Avatar.jsx

```diff
@@ -86,8 +86,8 @@ class Avatar extends React.Component {
     let avatarContent;
 
     if (image) {
-      avatarContent = generateImage(image, alt, isAriaHidden, AVATAR_VARIANTS.USER, this.handleFallback);
-    } else if (initials && (initials.length === 1 || initials.length === 2)) {
+      avatarContent = generateImage(image, alt, isAriaHidden, AVATAR_VARIANTS.USER, this.handleFallback, initials);
+    } else if (initials && initials.length <= 2) {
```

Comment:
Just curious, could these redundant checks be probably avoided if the default value of `initials` is set to an empty string instead of `undefined` and if the length of the `initials` prop is custom-validated during datatype validation?

Reply:
Redundant checks have been removed and validating this only at generateImagePlaceholder function : [here](https://github.com/cerner/terra-core/pull/2621/commits/565f1417aaeb24a8bcc372b80c65ba24ebe360ab)